bfe_modules/mod_tcp_keepalive/data_load.go

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2021 The BFE Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mod_tcp_keepalive
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+)
+
+/*
+{
+  "Version": "x",
+  "Config": {
+    "Product1": [{
+      "VipConf": ["1.1.1.1", "1.1.1.2"],
+      "KeepAliveParam": {
+		"Disable": false,
+        "KeepIdle" : 70,
+        "KeepIntvl" : 15,
+		"KeepCnt": 9
+      }
+    }]
+  }
+}
+*/
+// ProductRuleConf match the original tcp_keepalive.data
+type ProductRuleConf struct {
+	Version string
+	Config  map[string]ProductRulesFile
+}
+
+type ProductRulesFile []ProductRuleFile
+type ProductRuleFile struct {
+	VipConf        []string
+	KeepAliveParam KeepAliveParam
+}
+
+/*
+{
+  "Version": "x",
+  "Config": {
+    "Product1": {
+	  "1.1.1.1": {
+		"Disable": false,
+        "KeepIdle" : 70,
+        "KeepIntvl" : 15,
+		"KeepCnt": 9
+	  },
+	  "1.1.1.2": {
+		"Disable": false,
+        "KeepIdle" : 70,
+        "KeepIntvl" : 15,
+		"KeepCnt": 9
+	  }
+    }
+  }
+}
+*/
+// ProductRuleData contains data convert from ProductRuleConf
+type ProductRuleData struct {
+	Version string
+	Config  ProductRules
+}
+
+type KeepAliveParam struct {
+	Disable   bool
```

Comment:
Please add comments for meaning of the parameters?

Reply:
done